components/ExportEmails.vue

```diff
@@ -134,19 +135,28 @@ export default {
       selectedSpearator: ';',
       selectedRowSize  : 50,
       formatOptions,
-      exportOptions,
+      exportOptions    : [ ...baseExportOptions ],
       spearatorOptions,
       rowSizeOptions,
       textareaEmails   : [],
     };
   },
+  mounted,
   methods: {
-    onApply,
+    onEmailsExport,
     copy,
   },
 };
 
-function onApply() {
+function mounted() {
+  if (this.selectionType === 'organizations') {
+    this.exportOptions = baseExportOptions.map((e) => (e.text === 'Main + CCs address'
+      ? { ...e, text: 'Main + Focal Point' }
+      : e));
+  }
+}
```

Comment:
Make it simpler to understdand
```suggestion
function mounted() {
  if (this.selectionType === 'organizations') this.exportOptions.push({ value: 'MAINCCs', text: 'Main + Focal Point' });
  if (this.selectionType === 'contact')       this.exportOptions.push({ value: 'MAINCCs', text: 'Main + CCs address' });
}
```